Dockerfile

```diff
@@ -82,6 +83,9 @@ RUN groupadd --gid $USER_GID $USERNAME \
     --home-dir $HOMEDIR \
     --create-home $USERNAME
 
+RUN chgrp $USER /opt \
```

Comment:
Do the leading bullet item in the commit message still make sense? Their look like a left-over from rewording the commit message.

Reply:
Yes. 
The change above this line, set the environment to user ( line 75 ) and this reflects on the layers that depends on base now